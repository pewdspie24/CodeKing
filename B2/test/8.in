76
9910198121113107981081101041189911910311311011711811410611510011299122104121110991141111161181171021079811110197106991111079910210610411010710311910110298117
1021051151091079711998119115104108118110118102
10511411411511511711211012112299121105115121991031151209811697105100
119114105110105989897109117971131211001101181199799102112989897121109119103115101115108121105106109120106112110122999912011011912010511210110610210311310510012110612210910912111511411310510811411511198112106114107116102121105
100113121105104105113120101113110115102111101981011091021211131171141191049810699112116119121101108104111102120108113107971211011091091011221061081091101061081109911012210110711710311010410710911110298108118115109110112108105116115115112110102115119107107
99108112102109100116112114118110105981001221211021061209810511712010210311811710711397981019797991081171031081069911211310111010912010710311010011711612110610911812111911910410598
112109971201071051061011201161051041011151001221081171061181051161101169911310611798991031181099911811511110011610810912211410111010710410910212012010597115117121971159810411711199114106103104
11211410311810312110610612211411410612110811810511810112010310511712199106104110
11410010612010098118121108111991091141161081071001061121151061081131091101071041051011211061121069910210110911511011597105103113
10212110411010811611710410710810910310212011810310911210210010210111610210497115108100100117100112105104101107107122101116101113122120107113104105115116117104
116999711811611310211411211911512010898101110
119119107103113105111115100112106
1199910811711112110410111511210810610810711211912210810498105118114981099911210210710011211410010910911697111107109981131151051011201031031149910011710911598109119103121112989910611599106
118100103118106115103118105109110119111108109101108113113108115122100114104111121108116111105107118119120120108118100111105121104115121104103118121114117121101119100118110105107116971221029710811710211112211611411511699121109114113112109120103100108
99114119106109101115105111104119971111081041011091001039711212111011110810210210998113102113109991141091211181111011101059799107112
12111097107101112109119115114971141111081151121171041001151069910111110011811799113106104118122991071071171121229811311010411110197108100114118118112104107117115113
111118115991071171011061071091091161201061001161089810211211812011210310710510997110110120111971191201179710810411410410999112121111112102101105111113991221041011191111191071041091161089811710912197981011091141091041049997102104101101104111122111109108110
11810110111111011810610412111310011610512210597
122102104110120114104102116104119117107122101111105991101181129710410710812212197114105110100111111121122108112120119981181221071121091081139899113114112113115103107119116122108108115104112111113107120103116117116117111110115115112107118121
10310511210410411811311312297
109105104110119119105971061041061181211111141081001011171051211081051031031081181091151091031049711698121114109104119120981051031179710710610110211810411310310810711611911111010112210010310210410811511911410511611299122107109106101
1001059711799106102117108110106114110116114106101116114114107108118102117101981021211041201151131121119911710911910097107119102109105119118
101120109117120117110103112103116119103122104100114107108120971119910512210112012210711910711010812211910499991031211071011121001131229810011611710710510712010310011510911210511010012011010110911511311611610410010410012198118106108103
100111109111111102117108971041161181161171191111151021191071201211011171151221071041141021061079997109122100111122111981171131211111131081041121191119910112011010510399104122112116104110103115105971181001101161019710412010110498113116105
11011410011012099109105112114105112981041081031081141001091191031081161071171021081011021031181051061071091111221211051091071081161111101161021181151221021159711212111712211010199103105118103105115101103116971161111031051149810210610812011712010412297120110110
115100116119102105107117110103111119120120114109104991131011101181101199710611711610712011998119116116991091131071211131001111081121101091031189811511797108106105120108114119119
10111111810810211810210510810010410810411598991061101111081101221219811511810611610510499971081111041161181001109810511611498
9710598106107106115106111122101117119110109102102100113120104111100121114109110101103101120114981229710411512111610012010112098101117121108107110122121101112106114102971141031091111159710011511412210811211010711311510898108121120114120105
1211111151091171171111131141021051071221171131211081221021181139811212011210211510810312111010911712111711610411811811610211012112010610010498116100120109102120109114115103100118120106102109991141011141109998111115107116111114
1111149811010610411810412198991019711911711211610510010711310811511910111112111010799991091091179911311810497116122105119107113112103103105105109120115113121122118121112981031201141041149810311210697116106114104112
1091201071211061141201141161111131051081191011051201071129910510399100100107
10610411711211511698117110103102111111109111104122117100107108981081191191109910412111610410797115120110971181061219911610210810110412198119107118117112103104114111118111112122109113111116122971201101011079711210411511911512011212112199
1061131051161151049811710111512110711999103122107116
1131051159710511710511710310610697109106981021181161001191201001229811812210498112971151191141141021099811112111097113120112103116101107101111113115113110981181051081001001121131181021169711310699116115119106
109106119110118103989710011910410910412111712211099101119101118118108100110104101981191191221111061041151219810110310012112199117102121112111989910611710998100981169910611610211612212197117981141031001061051101171051121161011101181091059711210410110611710110011911812011910198101120115
1191171151031119710410810011312211911111598
11510910412211310211910110010710310910710111010611911211710798971111191141061051119910410810410711311997106108119109100115122105115106105114116106103
1111161021101091031121141161221111141041031041101151171221169710410412010411299109109104105100111109121102120107111120122109104112113102102111991141171081181091081011011119897117121119122113116114108112117114101115109107121109
114971129910811111198110122122103114107116101115111102109117102122100105
115119107981161001111101021221201091081131139810910311910798101115114118115116103121108107106119114107117112118102114120110117104111101102110
11911311710011011311610910411911611811010110911211112010611511210111912210911910910710110210911310810412110699102106114112121118121105107113122991191161211071039910810410712012210212012111511811910210710511111010212011098113120118
10310810812211797106101114971071071061181151181021019897114121119117
119109109121100109101114112108111118118106119103107121120107981061139710012098108119102104108114101103119110
121981019711810710111211411198106102117114104114114117104116103105117114110106109112108106100114102115114100103119991171101219911011099117117116971121091081001111021141011011191201061081191229898121117108110108100112117118105
121113981151039798999812211711310411110011010412211111412110999116107971071211141111181061181001061171061161071041001151021061021111211229811512098112114120105113102111971201021131079710110912211112110511111211210710010910798118
1221111061051121091169710910211310611211311611210811711911910311611310310910410111612099116113114120110101107110102115105971069812110610611610010210612210910210210010297118110109108109102
1111041021021111131061159810212211799114114109122989811510711210610898120121101112103119105101116104106121116111120103104114117981191021129810511811299120981171041159897105103112102
100105106112101118122118113109101100103122115116117102113122101101121105971171191211219910811812011299117119113122101105112121971051091171111219898118110105121115119117102109102107115991011011121041151059911810511210411198122111122
10398106117111101106121102102107111110981001091061229911510810399103119121113116101981209999105106100981171071121171151011211041171021021081091061221211141141221071191061161181031051159711312110710011598119120122102114120101
1091001151211221071151211131011059910810511610298112118107103107120110119104122991221021001111119910910911911811610711911011210610411112011599121100971111091031211211191221161021121171171061011101129811211910711311010812112110611999101118100106115107109100110
109104118107116109120104981041091141121201001089910210012010412211110398111110122115109122
112105981071221051079911999108121101119118991221089911910210711010410712110711910398981209810212210412010411811798114112105
119110121113113101108117104981051051051211021119710610611810010012211712111511612210710597116107103100111101119991181051201009710897101121120106108113106116106119104
1089910111098116106112117114119112116112117122122102117121117104100104991141141109710599979910310611610298105102104101111116110971169710210699103105115118111111106
11210498122118991029810411312012011510911611497121989912210010210112212210111210910710810310311511011110111411097112115121105120
1219811010797113106118114991141151171051229910311010811811411011610211310912210810299991021121031039710011911111911911811612111712210911498116114101110109981221211221071091071059912111997121116971051159812211399120102113116113981011041191081051039911597106102110122121114
100112103121115971011029811610811599101
1041209810411912212210010710110711510211811410511912010110611811411499111110108
104112109116111121101106113112107106121105981221141061011039710210010010612211612011211611312110611199103104108109
102107100100121104119120107118107119118991221171101011201221131201041111219911597114101110107103971049797105118122117114116109981021081101081091071191201031171041011121021171011149710597111112121104104102119121
1041051061031031001209811010111611810810912011611997114112106121
9711711211612210011210112111011811810112011610111311010810911899109122
1021181191079810611411898109109117103116118122105108117110118113991189710510611711697118118101120112114110122121105106100115107109117109122105102971001191149798105104115108121106100991201171009710810910510811997110103101113122
118102105104108121106108103103112116112989811110910312011811112010611810810711410910810710511310611410711210311799118109120120102106100120119104116114104100114118121109113100107
11511310011910710510611311411812010811010011010410010810211311610811011711811510410611199116114118110
10710311211712211311111710910611311512110510798105105981191019910612010111211611811011711310910211312110511898
1061051011091041101131061021201169911711899100109989811197114103109120103
115107112116116116971211191181221019910811611812210299
111121991089810012111511812010710411111311411511911910210710711799114106108105971041091081101171049711712012010610611212212199113103106103113112114971001141001161001051111061211019710210512210711311511910910411011010710811098117120105116102115117106102109118
971021141001131051101081089910211810810311910111110310711112012199114106109109122110119113106100112121102121121121121121102114109112
1211181061211041151021021131051031161131151091201041021001181221011099911110410011211710210098112114
1121151181171129910298102102101991021111091221011229710711711010112298103108118120122999710610999112105108115991021149710511410510411010110611710111097108120117103114971071219710711511499110121100122115116102111981091061171009711111510410411910110710798115105111107101119101108122111
1061209711597111971129710711811211512210697101108981111131051021091201031211161171131091031221071051051081049911410711410010310910610610012210912298111104121
110103119111971039811310010110912210611810699106116999711811710711198108119114116121105105109107117120114103114117117111120110108114991109911698111105109113117106109114
10111010697101991021221131131191051111001001161131011001151191119910411210510110911611710210910410811110111612111298112121
105106109108114121122115103105107
