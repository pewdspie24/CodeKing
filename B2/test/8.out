cebyqkblnhvcwgqnuvrjsdpczhyncrotvufkboeajcokcfjhnkgwefbu
fismkawbwshlvnvf
irrssupnyzcyisycgsxbtaid
wrinibbamuaqydnvwacfpbbaymwgseslyijmxjpnzccxnwxipejfgqidyjzmmysrqilrsobpjrktfyi
dqyihiqxeqnsfoebemfyqurwhbjcptwyelhofxlqkayemmezjlmnjlncnzekugnhkmofblvsmnplitsspnfswkk
clpfmdtprvnibdzyfjxbiuxfgvukqabeaaclugljcpqenmxkgndutyjmvywwhib
pmaxkijextihesdzlujvitntcqjubcgvmcvsodtlmzrenkhmfxxiasuyasbhuocrjgh
prgvgyjjzrrjylvivexgiuycjhn
rdjxdbvylocmrtlkdjpsjlqmnkhieyjpjcfemsnsaigq
fyhnltuhklmgfxvgmpfdfetfhaslddudpihekkzeteqzxkqhistuh
tcavtqfrpwsxlben
wwkgqiosdpj
wcluoyhespljlkpwzlhbivrbmcpfkdprdmmtaokmbqsiexggrcdumsbmwgypbcjscj
vdgvjsgvimnwolmelqqlszdrhoyltoikvwxxlvdoiyhsyhgvyruyewdvniktazfalufoztrstcymrqpmxgdl
crwjmesiohwaolhemdgapynolffmbqfqmcrmyvoeniackp
ynakepmwsrarolspuhdsjceodvucqjhvzckkupzbqnhoealdrvvphkusq
ovsckuejkmmtxjdtlbfpvxpgkimannxoawxualhrhmcpyopfeioqczhewowkhmtlbumyabemrmhhcafheehozomln
veeonvjhyqdtizia
zfhnxrhfthwukzeoicnvpahklzyarindooyzlpxwbvzkpmlqbcqrpqsgkwtzllshpoqkxgtutuonsspkvy
giphhvqqza
mihnwwiajhjvyorldeuiyligglvmsmghatbyrmhwxbiguakjefvhqglktwonezdgfhlswritpczkmje
diaucjfulnjrntrjetrrklvfuebfyhxsqpocumwdakwfmiwv
exmuxungpgtwgzhdrklxaocizexzkwknlzwhccgykepdqzbdtukikxgdsmpindxnemsqtthdhdybvjlg
domoofulahtvtuwosfwkxyeuszkhrfjkcamzdozobuqyoqlhpwocexnigchzpthngsiavdnteahxehbqti
nrdnxcmipripbhlglrdmwgltkuflefgvijkmozyimkltontfvszfsapyuznecgivgisegtatogirbfjlxuxhzaxnn
sdtwfikungowxxrmhcqenvnwajutkxwbwttcmqkyqdolpnmgvbsualjixlrww
eovlfvfildhlhsbcjnolnzybsvjtihcalohtvdnbitrb
aibjkjsjozeuwnmffdqxhodyrmnegexrbzahsytdxexbeuylknzyepjrfargmosadsrzlpnkqslblyxrxi
yosmuuoqrfikzuqylzfvqbpxpfslgynmuyuthvvtfnyxjdhbtdxmfxmrsgdvxjfmcrerncbosktor
orbnjhvhybceawuptidkqlsweoynkccmmucqvhatziwkqpggiimxsqyzvypbgxrhrbgpjatjrhp
mxkyjrxrtoqilweixkpcigcddk
jhupstbungfoomohzudklblwwnchythkasxnavjyctflehybwkvupghrovopzmqotzaxnekaphswsxpyyc
jqitshbuesykwcgzkt
qisaiuiugjjamjbfvtdwxdzbvzhbpaswrrfmboynaqxpgtekeoqsqnbvilddpqvftaqjctswj
mjwnvgbadwhmhyuzncewevvldnhebwwzojhsybegdyycufypobcjumbdbtcjtftzyaubrgdjinuiptenvmiaphejuedwvxwebexs
wusgoahldqzwosb
smhzqfwedkgmkenjwpukbaowrjiochlhkqwajlwmdszisjirtjg
otfnmgprtzorhghnsuztahhxhpcmmhidomyfxkoxzmhpqffocrulvmleeobauywzqtrlpuresmkym
rapcloobnzzgrktesofmufzdi
swkbtdonfzxmlqqbmgwkbesrvstgylkjwrkupvfrxnuhoefn
wqudnqtmhwtvnempoxjspewzmwmkefmqlhyjcfjrpyvyikqzcwtykgclhkxzfxysvwfkionfxnbqxv
gllzuajerakkjvsvfebarywu
wmmydmerplovvjwgkyxkbjqadxblwfhlregwn
ybeavkeprobjfurhrruhtgiurnjmpljdrfsrdgwcunycnncuutapmldofreewxjlwzbbyulnldpuvi
yqbsgabcbzuqhodnhzorymctkakyrovjvdjujtkhdsfjfoyzbsxbprxiqfoaxfqkaemzoyioppkdmkbv
zojipmtamfqjpqtpluwwgtqgmhetxctqrxneknfsiajbyjjtdfjzmffdfavnmlmf
ohffoqjsbfzucrrmzbbskpjlbxyepgwiethjytoxghrubwfpbivpcxbuhsbaigpf
dijpevzvqmedgzstufqzeeyiauwyyclvxpcuwqzeipyaimuoybbvniyswufmfksceephsicviphobzoz
gbjuoejyffkonbdmjzcslgcgwyqtebxccijdbukpuseyhufflmjzyrrzkwjtvgisaqykdsbwxzfrxe
mdsyzksyqeiclitfbpvkgkxnwhzczfdoocmmwvtkwnpjhoxscydaomgyywztfpuujenpbpwkqnlyyjwcevdjskmdn
mhvktmxhbhmrpxdlcfdxhzogbonzsmz
pibkzikcwclyewvczlcwfknhkykwgbbxbfzhxhvubrpi
wnyqqeluhbiiiyfoajjvddzuystzkiatkgdoewcvixdalaeyxjlqjtjwh
lcenbtjpurwptpuzzfuyuhdhcrrnaicacgjtfbifheotnatafjcgisvooj
phbzvcfbhqxxsmtraybczdfezzepmklggsnoernapsyix
ybnkaqjvrcrsuizcgnlvrntfqmzlfccfpggadwowwvtyuzmrbtrenmbzyzkmkicywaytaisbzqcxfqtqbehwligcsajfnzyr
dpgysaefbtlsce
hxbhwzzdkeksfvriwxejvrrconl
hpmtoyejqpkjyibzrjegafddjztxptqyjocghlm
fkddyhwxkvkwvczunexzqxhoycsarenkgahaaivzurtmbflnlmkwxguhepfueraiaopyhhfwy
hijggdxbnetvlmxtwarpjy
auptzdpeynvvexteqnlmvcmz
fvwkbjrvbmmugtvzilunvqcvaijutavvexprnzyijdskmumzifadwrabihslyjdcxudalmilwangeqz
vfihlyjlggptpbbomgxvoxjvlkrmlkiqjrkpgucvmxxfjdxwhtrhdrvymqdk
sqdwkijqrvxlndnhdlfqtlnuvshjoctrvn
kgpuzqoumjqsyikbiibwecjxeptvnuqmfqyivb
jiemhnqjfxtcuvcdmbboargmxg
skptttaywvzecltvzfc
oyclbdysvxkhoqrswwfkkucrjliahmlnuhauxxjjpzycqgjgqpradrdtdiojyeafizkqswmhnnklnbuxitfsujfmv
afrdqinllcfvlgweogkoxycrjmmznwqjdpyfyyyyyfrmp
yvjyhsffqigtqsmxhfdvzemcohdpufdbpr
psvupcfbffecfomzezakunezbglvxzcajmcpilscfrairihnejuenalxugrakyaksrcnydzstfobmjudaoshhwekkbsiokewelzo
jxasaoapakvpszjaelboqifmxgytuqmgzkiilhcrkrdgmjjdzmzbohy
ngwoagbqdemzjvjcjtcavukoblwrtyiimkuxrgruuoxnlrcnctboimqujmr
enjaecfzqqwioddtqedswochpiemtufmhloetypbpy
ijmlryzsgik
